src/components/summary/_macro.njk

```diff
@@ -51,6 +51,9 @@
                                                                 "iconType": rowItem.iconType
                                                             })
                                                         -}}
+                                                        {% if rowItem.iconVisuallyHiddenText %}
+                                                            <span class="ons-u-vh">Section completed</span>
+                                                        {% endif %}
```

Comment:
Is there a need for this? We asked Joe and the consensus was for standardisation, we would want the displayed text to be hard coded. Are there scenarios where the check icon is used in the table to not mean section completed?

Reply:
We discussed this yesterday during stand up and it was agreed to set the text ourselves to avoid any inconsistencies amongst services.

Reply:
This needs to be configurable because it is not necessarily always going to need to be "Section completed", maybe it will need to be something slightly different in this case even. Maybe there is a need to use a different icon for another reason which we allow, but the text would still be "Section completed". Hardcoding it like this would also mean that there would be no way of translating it, it would always be in English. We don't want to be hardcoding content like this, that should be left to the service.

Reply:
Yeah good points. These weren't raised yesterday and I agree with them. I will revert the change